crates/rust-analyzer/src/main_loop.rs

```diff
@@ -787,6 +847,28 @@ impl GlobalState {
         }
     }
 
+    fn handle_discover_msg(&mut self, message: flycheck::DiscoverProjectMessage) {
+        match message {
+            flycheck::DiscoverProjectMessage::Finished(output) => {
+                self.report_progress("Buck", Progress::End, None, None, None);
```

Comment:
`"Buck"` shouldn't be hard-coded here; it should either come from configuration or the crate's label.

(hard blocker again, IMO.)